### PR TITLE
Supporting hierarchical directory structure for bricks

### PR DESCRIPTION
**Pull request by gaurav36**
While creating volume using glusterd2 it should support creation of
hirearchical directory stracture, if brick present in hierarchical
directory.

Issue #68

Signed-off-by: Gaurav Kumar Garg <garg.gaurav52@gmail.com>

### CODE REVIEW

**Comment by atinmu** [comment]
@gaurav36 : Please add a test in utils/utils_test.go covering this change.

File: utils/utils.go [hunk 1]
```diff
@@ -12,6 +11,8 @@ import (
 	"strings"
 	"syscall"
 
+	"golang.org/x/sys/unix"
```
Comment (atinmu) [inline utils/utils.go]: Why do you need this package to be imported?

Comment (gaurav36) [inline utils/utils.go]: I have not include this package this package included automatically while saving file in vim-go. for more information about this package you can follow https://godoc.org/golang.org/x/sys/unix

Comment (gaurav36) [inline utils/utils.go]: or you can see the use case on    line number 28..31 in this file (utils/utils.go)

**Pull request merged**